bloptools/gp/kernels.py

```diff
@@ -0,0 +1,108 @@
+import torch
+import gpytorch
+import numpy as np
+import scipy as sp
+
+#gpytorch.linear_operator.operators import LinearOperator, MatmulLinearOperator, RootLinearOperator
+
+class LatentMaternKernel(gpytorch.kernels.Kernel):
+
+    def __init__(
+        self,
+        n_dof,
+        length_scale_bounds,
+        off_diag=True,
+        **kwargs,
+    ):
+        super(LatentMaternKernel, self).__init__(**kwargs)
```

Comment:
I don't think the arguments are _required_ for the `super()` call. I think it was needed in the past with Py2.x.